plotly/plotly/plotly.py

```diff
@@ -1493,6 +1493,42 @@ def get_grid(grid_url, raw=False):
         return json_res
 
 
+def create_animations(figure, filename=None, sharing='public'):
+    """
```

Comment:
And we should describe how this is similar and dissimilar to `plotly.plotly.plot`. Includes things like:
- Folders aren't supported
- Overwrite isn't supported
- Animations via frames are supported
- There might be some other things that I'm forgetting about.